sinon/index.d.ts

```diff
@@ -109,6 +109,7 @@ declare namespace Sinon {
         callsArgOnAsync(index: number, context: any): SinonStub;
         callsArgWithAsync(index: number, ...args: any[]): SinonStub;
         callsArgOnWithAsync(index: number, context: any, ...args: any[]): SinonStub;
+        callsFake(func: any): SinonStub;
```

Comment:
Do you have a more accurate type for `func`? I'd pick `Function` or `(...args: any[] => void` at the very least.